Source/VisualRecognitionV3/VisualRecognition+CoreML.swift

```diff
@@ -128,29 +128,40 @@ extension VisualRecognition {
     /**
      Classify an image using a Core ML model from the local filesystem.
 
-     - parameter image: The image to classify.
+     - parameter imageFile: The image to classify.
      - parameter classifierIDs: A list of the classifier ids to use. "default" is the id of the
        built-in classifier.
      - parameter threshold: The minimum score a class must have to be displayed in the response.
      - parameter failure: A function executed if an error occurs.
      - parameter success: A function executed with the image classifications.
      */
     public func classifyWithLocalModel(
-        image: UIImage,
+        imageFile: URL,
         classifierIDs: [String] = ["default"],
         threshold: Double? = nil,
         failure: ((Error) -> Void)? = nil,
         success: @escaping (ClassifiedImages) -> Void)
     {
-        // convert UIImage to Data
-        guard let image = UIImagePNGRepresentation(image) else {
-            let description = "Failed to convert image from UIImage to Data."
-            let userInfo = [NSLocalizedDescriptionKey: description]
-            let error = NSError(domain: self.domain, code: 0, userInfo: userInfo)
+        // convert imageFile to Data
+        var imageData: Data
+        do {
+            imageData = try Data(contentsOf: imageFile)
+        } catch {
             failure?(error)
             return
         }
 
+        classifyWithLocalModel(imageData: imageData, classifierIDs: classifierIDs, threshold: threshold,
+                               failure: failure, success: success)
+    }
+
+    internal func classifyWithLocalModel(
```

Comment:
Any reason not to expose this publicly? This would enable low-latency use-cases on macOS without needing to write to file (e.g. streaming from web cam)

Reply:
I think we should pick one flavor to expose -- let's not overcomplicate the API.  So if you want to expose the version that accepts a `Data`, let's eliminate the one that accepts a `URL`.  Reasonable?

Reply:
Does is make sense to mirror `VNImageRequestHandler` and support `CGImage`, `CIImage`, `CVPixelBuffer`, `Data` and `URL`?

Reply:
Only if there is a compelling use case for it.